### PR TITLE
historical_founder_service

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: business_register/converter/history.py [hunk 1]
```diff
@@ -71,3 +71,38 @@ def save_to_db(self, records):
...
+            try:
+                company_exists = Company.objects.filter(edrpou=edrpou).first()
+                founder.company = company_exists
+            except AttributeError:
+                continue
+            try:
+                founder.id = Signer.objects.filter(company=company_exists).first().id
+            except AttributeError:
+                founder.id = 0 #for changed records that can't be assigned to existing company
+            founder.history_type = '~'
```
Comment (criticus) [inline business_register/converter/history.py:103]: what does tilde mean here?

Comment (roman-tiukh) [reply business_register/converter/history.py:103]: This is type `update`.
I think you need provide constants for history types.

Comment (Iva-khar) [reply business_register/converter/history.py:103]: This Class for only one time using

Comment (roman-tiukh) [reply business_register/converter/history.py:103]: add comment at least

**Pull request merged**